_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -50787,12 +50768,10 @@
 /turf/open/floor/plasteel,
 /area/engine/engineering)
 "cjb" = (
-/obj/structure/cable{
-	d1 = 2;
-	d2 = 4;
-	icon_state = "2-4"
+/obj/effect/turf_decal/delivery,
+/turf/open/floor/plasteel{
+	name = "floor"
```

Comment:
jesus christ they are, reeeeeeeee.

thanks for pointing that out, adding that to my cleanup PR lol